doc/api/deprecations.md

```diff
@@ -684,6 +684,13 @@ difference is that `querystring.parse()` does url encoding:
 { '%E5%A5%BD': '1' }
 ```
 
+<a id="DEP0077"></a>
+### DEP0077: Loading files without a known file extension 
```

Comment:
The numbers should be assigned only during the commit landing process.

Reply:
We landed one with `XX` already, unsure how to ensure that gets done? Who assigns these/when?

Reply:
@bmeck Whoever is landing the commit, they will assign the latest available number usually.